internal/clusterfeature/features/vault/common.go

```diff
@@ -0,0 +1,27 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vault
+
+const (
+	featureName             = "vault"
+	vaultWebhookReleaseName = "vault-secrets-webhook"
+	kubeSysNamesapce        = "kube-system"
+	vaultAddressEnvKey      = "VAULT_ADDR"
+	rolePath                = "auth/kubernetes/role/pipeline-webhook"
+	roleName                = "pipeline-webhook"
+	authMethodType          = "kubernetes"
+	authMethodPath          = "kubernetes"
```

Comment:
This is too general, I think we need to find another name, like `kubernetes-cluster-auth` or something like that. Let's discuss that offline first.